src/db/index.ts

```diff
@@ -0,0 +1,39 @@
+import { DataTypes } from 'sequelize';
+import UsersAttributes from '../types/usersAttributes';
```

Comment:
please import it from `/types` index file

Reply:
done